javascript/ql/src/Security/CWE-843/examples/TypeConfusionThroughParameterTampering.js

```diff
@@ -4,8 +4,8 @@ var app = require("express")(),
 app.get("/user-files", function(req, res) {
   var file = req.param("file");
   if (file.indexOf("..") !== -1) {
-    // BAD
-    // forbid paths outside the /public directory
+    // BAD: we forbid relative paths that contain ..
```

Comment:
The `BAD` refers to the code example being a "good" or "bad"; it's not about the fact that the incoming request is classified as a "bad request" or not. It looks like it was accidentally copy/pasted into the fixed example without changing it to `GOOD`.

The rest of the comment is really just part of code snippet so should be on a separate line
```suggestion
    // BAD
    // we forbid relative paths that contain ..
```